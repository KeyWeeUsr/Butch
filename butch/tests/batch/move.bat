:: run from test.py cwd assuming repo root
cd butch
cd tests
echo %errorlevel%
:: FNF
move nonexisting nonexisting
echo %errorlevel%

:: multi-FNF
move nonexisting nonexisting nonexisting
echo %errorlevel%

:: FNF to explicit dir
move nonexisting nonexisting nonexisting\
echo %errorlevel%

:: file renaming
move existing nonexisting
echo %errorlevel%
type nonexisting
echo %errorlevel%
del nonexisting
echo %errorlevel%

:: move to folder by wildcard
type first1
echo %errorlevel%
type first2
echo %errorlevel%

:: move multiple files to single file
move first* nonexisting
echo %errorlevel%
:: error

mkdir existing
echo %errorlevel%
move first* nonexisting\
echo %errorlevel%
move first* existing
echo %errorlevel%
cd existing
del first1
echo %errorlevel%
del first2
echo %errorlevel%
cd ..
rmdir existing
echo %errorlevel%
cd ..
cd ..
