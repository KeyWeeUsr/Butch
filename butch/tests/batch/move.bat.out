0
<stderr> The system cannot find the file specified.
1
<stderr> The syntax of the command is incorrect.
1
<stderr> The syntax of the command is incorrect.
1
0
hello
0
0
hello
0
world
0
<stderr> Cannot move multiple files to a single file.
1
0
<stderr> Cannot move multiple files to a single file.
1
0
0
0
0
